packages/vuetify/src/components/VTabs/VTabs.sass

```
@@ -90,6 +90,18 @@
   .v-tabs-slider-wrapper + *
     margin-left: auto
 
+  +rtl()
+    .v-tabs-bar__content > *:last-child
+      margin-right: 0
+      margin-left: auto
+
+    .v-tabs-bar__content > *:first-child:not(.v-tabs-slider-wrapper)
+      margin-left: 0
+      margin-right: auto
+
+    .v-tabs-slider-wrapper + *
+      margin-left: 0
+      margin-right: auto
 .v-tabs--fixed-tabs
```

Space after block please
```suggestion
 .v-tabs--fixed-tabs

```